README.md

```diff
@@ -80,6 +83,7 @@ const PeopleSearch = () = {
 - `pollInterval` - How often to re-request updated data. Pass 0 to disable polling (the default behaviour).
 - `payload` - A data object to send in the request. If we are performing a GET request, it is appended into the querystring (e.g. `?keywords=hello`). If it is a POST request it is sent in the request body as JSON.
 - `method` - Set the request type, either `get` or `post`. (defaults to `get`)
+- `changed`: A function that is called if the data actually changed during the request. If this is specified, use-api does extra checking and compares old and new data. If data does not change, data remains unchanged. Please note, this may have performance repercussions if the data is large as it performs a deep comparison between new and old data to determine if they are equivalent.
```

Comment:
```suggestion
- `changed`: A function that is called if the data actually changed during the request. If this is specified, use-api does extra checking and compares old and new data. If data does not change, new data is not propagated and a redraw is saved. Please note, this may have performance repercussions if the data is large as it performs a deep comparison between new and old data to determine if they are equivalent.
```